authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/services/ConnectorService.kt

```diff
@@ -114,6 +114,7 @@ class ConnectorService(
         val c = Tables.CONNECTOR
         dsl.update(c)
             .setNull(c.PROVIDER_ORGANIZATION_ID)
+            .setNull(c.CREATED_BY)
```

Comment:
Is there a scenario where `CREATED_BY` and `PROVIDER_ORGANIZATION_ID` are not related to each other (e.g. `CREATED_BY` not part of organization with `PROVIDER_ORGANIZATION_ID`)? If yes, you would lose the `CREATED_BY` info even if it is unrelated and could stay.

Reply:
No, such a case is currently not possible. The creating user is always part of the organization that registers the connector. In the case of provided connectors, this is the provider organization.